README.md

```diff
@@ -175,7 +175,7 @@ services:
       optionalStuff: foo
 ```
 
-In this sample, Secretless Broker is configured to connect to a service named `postgres-db`. Clients send connection requests to this service via localhost using port 5432. This service uses the `pg` protocol, which indicates that the PostgreSQL service connector should process requests that come in via this port. Credentials from this connection will be retrieved from multiple sources; the `address` is given as a string value, the `password` will be retrieved from the Conjur variable with ID `id-of-secret-in-conjur`, and the `username` is retrieved from the environment variable named `username`.
+In this sample, Secretless Broker is configured to connect to a service named `postgres-db`. Clients send connection requests to this service via localhost using the default port 5432. This service uses the `pg` protocol, which indicates that the PostgreSQL service connector should process requests that come in via this port. Credentials from this connection will be retrieved from multiple sources; the `host` is given as a literal string value, the `password` will be retrieved from the Conjur variable with ID `id-of-secret-in-conjur`, and the `username` is retrieved from the environment variable named `username`.
```

Comment:
Not related to this paragraph, but there are other example snippets on this page that use the provider v1 syntax, and those should be updated.  Do a search for `provider:`